sky/global_user_state.py

```diff
@@ -594,19 +608,23 @@ def get_cluster_from_name(
             'storage_mounts_metadata':
                 _load_storage_mounts_metadata(storage_mounts_metadata),
             'cluster_ever_up': bool(cluster_ever_up),
+            'config_hash': config_hash,
         }
         return record
     return None
 
 
 def get_clusters() -> List[Dict[str, Any]]:
     rows = _DB.cursor.execute(
-        'select * from clusters order by launched_at desc').fetchall()
+        'select name, launched_at, handle, last_use, status, autostop, '
+        'metadata, to_down, owner, cluster_hash, storage_mounts_metadata, '
+        'cluster_ever_up, config_hash from clusters order by launched_at desc'
```

Comment:
can we have a constants for all the columns, see: https://github.com/skypilot-org/skypilot/blob/master/sky/jobs/state.py#L131-L164

Reply:
Sure, I'll do it in a separate PR. I made this same change in #4332 so this is already in master.